magicctapipe/scripts/lst1_magic/semi_automatic_scripts/merge_stereo.py

```diff
@@ -0,0 +1,141 @@
+"""
+This scripts merges LST DL1Stereo subruns into runs
+
+Usage:
+$ python merge_stereo.py (-c config_file.yaml)
+"""
+import argparse
+import glob
+import logging
+import os
+from pathlib import Path
+
+import joblib
+import numpy as np
+import yaml
+
+from magicctapipe import __version__
+from magicctapipe.scripts.lst1_magic.semi_automatic_scripts.clusters import (
+    rc_lines,
+    slurm_lines,
+)
+
+__all__ = ["MergeStereo"]
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
+
+
+def MergeStereo(target_dir, env_name, source, cluster):
+    """
+    This function creates the bash scripts to run merge_hdf_files.py in all DL1Stereo subruns.
+
+    Parameters
+    ----------
+    target_dir : str
+        Path to the working directory
+    env_name : str
+        Name of the environment
+    source : str
+        Name of the target
+    cluster : str
+        Cluster system
+    """
+
+    process_name = source
+    stereo_DL1_dir = f"{target_dir}/v{__version__}/{source}"
+    listOfNightsLST = np.sort(glob.glob(f"{stereo_DL1_dir}/DL1Stereo/*"))
+    if cluster != "SLURM":
+        logger.warning(
+            "Automatic processing not implemented for the cluster indicated in the config file"
+        )
+        return
+    for nightLST in listOfNightsLST:
+        night = nightLST.split("/")[-1]
+        stereoMergeDir = f"{stereo_DL1_dir}/DL1Stereo/{night}/Merged"
+        os.makedirs(f"{stereoMergeDir}/logs", exist_ok=True)
+
+        if len(glob.glob(f"{nightLST}/dl1_stereo*.h5")) < 1:
+            continue
+
+        slurm = slurm_lines(
+            queue="short",
```

Comment:
did you check the memory requirements from those merging jobs, I think they will need much less than the default of 5G

Reply:
not yet. I will check them with job_accounting